graph/graph_windows.go

```diff
@@ -114,7 +119,29 @@ func (graph *Graph) storeImage(img *Image, layerData archive.ArchiveReader, root
 
 		// Store the layer. If layerData is not nil, unpack it into the new layer
 		if layerData != nil {
-			if img.Size, err = graph.driver.ApplyDiff(img.ID, img.Parent, layerData); err != nil {
+			// this is saving the tar-split metadata
```

Comment:
@calavera The right approach here is to make the WindowsGraphDriver not be a snowflake. I am not sure how this even got merged. After an examination of the diff between `graph_windows.go` and `graph_unix.go`, there are no real, system specific differences. The real issue was a lack of imagination in the implementation.